beetmoverscript/templates/firefox_partner.yml

```diff
@@ -0,0 +1,42 @@
+---
+metadata:
+    name: "Beet Mover Manifest"
+    description: "Maps Firefox artifacts to pretty names for en-US"
+    owner: "release@mozilla.com"
+
+s3_bucket_path: pub/firefox/candidates/{{ version }}-candidates/build{{ build_number }}/
+
+mapping:
+{% for locale in ['en-US'] %}
+  {{ locale }}:
+  {% if platform in ["linux-i686", "linux-x86_64"] %}
+    target.tar.bz2:
+      s3_key: {{ platform }}-EME-free/{{ locale }}/firefox-{{ version }}.tar.bz2
```

Comment:
Ah, looks like this is currently the public template only. This works, but we'll need to either support both the private and public partners in the same template(s), or have separate partners behaviors and templates.